rust/operator-binary/src/crd/catalog/mod.rs

```diff
@@ -88,6 +91,6 @@ mod tests {
 
     #[test]
     fn test_crd_generation() {
-        TrinoCatalog::crd();
+        v1alpha1::TrinoCatalog::crd();
```

Comment:
This should use the `crd::TrinoCatalog::merged_crd` method to generate the whole crd, not just one version.

Reply:
[done](https://github.com/stackabletech/trino-operator/pull/711/commits/cdc58bc54a61a0faba1375af2b0d90173c5a036d)

Reply:
[done](https://github.com/stackabletech/trino-operator/pull/711/commits/1ed647756527238b874abc230ba3d57d7ba61407)